.eslintrc.js

```diff
@@ -1,7 +1,28 @@
 module.exports = {
   root: true,
-  // This tells ESLint to load the config from the package `eslint-config-custom`
-  extends: ["custom"],
+
+  extends: ["eslint:recommended"],
```

Comment:
Why did we need to update .eslintrc here?

The "custom" eslint config in packages is there so we can reuse it across multiple project. Common eslint rules should probably go there.

Reply:
But you're right, the custom config is for next project, might not be suitable for everything. We need to think what to do with those configs.

Reply:
The custom config was throwing an error while running the Super Linter. 
I think the probable cause is that this Linter does require the `npm install` step (Assuming the "custom" config requires node install first). It pulls the required modules at run time from docker, I guess. 

So had to resort to the recommended lint config.

Reply:
We can put in multiple configs there as well. Let me see if there is a way to include both custom and recommended and have Next projects run with custom and the rest with recommended.